docs/docs/integrations/libraries/looker/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Dagster & Looker (Component)
+sidebar_label: Looker
```

Comment:
You'll want to move the existing `looker.md` file:

* https://github.com/dagster-io/dagster/blob/925210d0504164529d3cd8e6589ad6fb905c11c3/docs/docs/integrations/libraries/looker.md

to a `looker/looker-pythonic.md` file and update the sidebar label.